### ISSUE TITLE
AttributeError: 'dict' object has no attribute 'iteritems'

### ISSUE THREAD

**Issue by diwakergupta** [mesozoic/gmail-yaml-filters#6]
`gmail-yaml-filters` is failing for me, with any options, with the samples included in the documentation. 

```bash
$ python3 -V
Python 3.6.5

$ pip3 list | grep yaml
gmail-yaml-filters       0.7.2

$ $ cat test-filters.yaml
# Simple example
-
  from: googlealerts-noreply@google.com
  label: news
  not_important: true

$ gmail-yaml-filters test-filters.yaml
Traceback (most recent call last):
  File "/usr/local/bin/gmail-yaml-filters", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/gmail_yaml_filters/main.py", line 595, in main
    ruleset = RuleSet.from_object(rule for rule in data if not rule.get('ignore'))
  File "/usr/local/lib/python3.6/site-packages/gmail_yaml_filters/main.py", line 487, in from_object
    return cls.from_iterable(obj, base_rule=base_rule)
  File "/usr/local/lib/python3.6/site-packages/gmail_yaml_filters/main.py", line 516, in from_iterable
    ruleset.update(cls.from_object(data, base_rule=base_rule))
  File "/usr/local/lib/python3.6/site-packages/gmail_yaml_filters/main.py", line 485, in from_object
    return cls.from_dict(obj, base_rule=base_rule)
  File "/usr/local/lib/python3.6/site-packages/gmail_yaml_filters/main.py", line 503, in from_dict
    new_rule = Rule(data, base_rule=base_rule)
  File "/usr/local/lib/python3.6/site-packages/gmail_yaml_filters/main.py", line 337, in __init__
    self.update(data)
  File "/usr/local/lib/python3.6/site-packages/gmail_yaml_filters/main.py", line 353, in update
    for key, value in dict(data).iteritems():
AttributeError: 'dict' object has no attribute 'iteritems'
```

**Comment by mesozoic** [comment]
Thanks for pointing this out! I'm sorry for the delay in responding, but it turns out I needed to tune my GitHub notification settings.

I haven't updated for Python 3 compatibility yet. This library still only supports Python 2.7 (and is tagged as such in `setup.py`). I don't imagine it will take a ton of work, but I haven't got around to it. I'm open to pull requests if anyone wants to take a stab at it before I have a chance :)

**Comment by mesozoic** [comment]
I've fixed this with v0.8, which is tested on py36 and py37.
